manim/config/utils.py

```diff
@@ -95,6 +95,7 @@ def determine_quality(args):
 class ManimConfig(MutableMapping):
 
     _OPTS = {
+        "assets_dir",
```

Comment:
You also need to add two lines to the `manim/config/default.cfg` file. One in the CLI section and one in the custom_folders section.

Reply:
``assets_dir`` isn't supposed to be used for output folder, but rather a place for input folder. I added it to the CLI default section, but for custom_folder it seems these are all output related and ``assets_dir`` shouldn't be modified there

Reply:
Fair.